client/css/style.css

```diff
@@ -176,6 +176,126 @@ button {
 	color: rgba(0, 0, 0, .35) !important;
 }
 
+/* Icons */
+
+#viewport .lt:before,
+#viewport .rt:before,
+#chat button.menu:before,
+#sidebar .chan:before,
+#chat .title:before,
+#footer .icon,
+#chat .count:before,
+#settings #play:before,
+#form #submit:before,
+#chat .invite .from:before,
+#chat .join .from:before,
+#chat .kick .from:before,
+#chat .part .from:before,
+#chat .quit .from:before,
+#chat .topic .from:before,
+#chat .mode .from:before,
+#chat .ctcp .from:before,
+#chat .whois .from:before,
+#chat .nick .from:before,
+#chat .action .from:before,
+.context-menu-item:before {
+	font: normal normal normal 14px/1 FontAwesome;
+	font-size: inherit; /* Can't have font-size inherit on line above, so need to override */
+	-webkit-font-smoothing: antialiased;
```

Comment:
If we could put `subpixel-antialiased` there instead it would be pretty great. On my computer it makes a small but noticable difference by making the icon slighly more sharp.

![capture d ecran_2016-07-16_14-31-52](https://cloud.githubusercontent.com/assets/5481612/16896509/56d865d4-4b63-11e6-9a51-a29e9108d382.png)